crate_universe/extension.bzl

```diff
@@ -40,8 +40,28 @@ _generate_repo = repository_rule(
     ),
 )
 
-def _generate_hub_and_spokes(module_ctx, cargo_bazel, cfg, annotations):
-    cargo_lockfile = module_ctx.path(cfg.cargo_lockfile)
+def _annotations_for_repo(module_annotations, repo_specific_annotations):
```

Comment:
nit: Do you wanna document this one as well 😅

Reply:
Sure, done